lib/view_component/preview.rb

```diff
@@ -62,6 +63,16 @@ def layout(layout_name)
         @layout = layout_name
       end
 
+      # Returns the relative path (from preview_path) to the preview example template if the example and template exists
+      def preview_example_template_path(example)
+        return nil unless example_exists?(example)
+
+        path = Dir["#{preview_path}/#{preview_name}_preview/#{example}.html.erb"].first
```

Comment:
@juanmanuelramallo doesn't this line couple us to ERB specficially?

Reply:
Hmm you are right, I completely forgot about this sorry. 

Then I think we should refactor this a bit more and make the examples with templates not use the body-less methods from the preview object, and just look into the correct folder for any template there. If there's any then show those examples (with whatever handler they might have). 

Does that make sense?

Reply:
Makes sense to me 👍